src/main/resources/okta/js/applications.js

```diff
@@ -0,0 +1,618 @@
+;(function( $ ) {
+    $.fn.zclip = function() {
+        if(typeof ZeroClipboard == 'function'){
+            var client = new ZeroClipboard( this );
+            client.on( "ready", function( readyEvent ) {
+              client.on( "aftercopy", function( event ) {
+                var target = $(event.target);
+                target.attr("title","Copied!")
+                target.tooltip('enable');
+                target.tooltip("show");
+                target.tooltip('disable');
+              });
+            });
+        }else{
+            console.warn('Warning : Dependency missing - ZeroClipboard Library');
+        }
+        return this;
+    };
+}( jQuery ));
+
+
+var GrantTypes = function (available) {
+    //order will be preserved in the response map
+    this.config = {
+        "authorization_code":"Code",
+        "implicit":"Implicit",
+        "refresh_token":"Refresh Token", 
+        "password":"Password", 
+        "iwa:ntlm":"IWA-NTLM", 
+        "client_credentials":"Client Credential", 
+        "urn:ietf:params:oauth:grant-type:saml2-bearer":"SAML2",
+    }
+
+    this.available = {};
+    for(var i=0;i < available.length;i++){
+        this.available[available[i]] = this.config[available[i]];
+    }
+};
+
+GrantTypes.prototype.getMap = function(selected){
+    var grants = [];
+    if(selected !=undefined)
+        grants = selected.toString().split(" ");
+    var map = [];
+    for(var grant in this.available){
+        var selected = grants.indexOf(grant) > -1;
+        map.push({ key: grant , name:this.available[grant], "selected" : selected});
+    }
+
+    return map;
+};
+
+;(function ( $, window, document, undefined ) {
+
+    var pluginName = "codeHighlight";
+
+    // The actual plugin constructor
+    function Plugin( element, options ) {
+        this.element = $(element);
+        this._name = pluginName;
+        this.init();
+    }
+
+    Plugin.prototype = {
+
+        init: function() {
+            this.editor = CodeMirror.fromTextArea(this.element[0], {
+              mode:  "shell",
+              readOnly: true,
+              lineWrapping: true
+            });
+        },
+
+        refresh: function(){
+            this.editor.refresh();
+        }
+
+
+    };
+
+    // A really lightweight plugin wrapper around the constructor,
+    // preventing against multiple instantiations
+    $.fn[pluginName] = function ( options ) {
+        return this.each(function () {
+            if (!$.data(this, "plugin_" + pluginName)) {
+                $.data(this, "plugin_" + pluginName,
+                new Plugin( this, options ));
+            }
+        });
+    };
+
+})( jQuery, window, document );
+
```

Comment:
There are several extra lines. Please find and remove those.

Reply:
Fixed